common/sai_npu.py

```diff
@@ -207,7 +206,8 @@ def set_sku_mode(self, sku):
         # Remove existing ports
         num_ports = len(self.dot1q_bp_oids)
         for idx in range(num_ports):
-            self.remove_vlan_member(self.default_vlan_oid, self.dot1q_bp_oids[idx])
+            if self.get_vlan_member(self.default_vlan_oid, self.dot1q_bp_oids[idx]):
```

Comment:
I'd rewrite this as follows
```
oid =  self.get_vlan_member(self.default_vlan_oid, self.dot1q_bp_oids[idx])
if oid:
    self.remove(oid)
```
This will work faster then `remove_vlan_member`